pandas/tests/io/test_pickle.py

```diff
@@ -248,7 +219,8 @@ def test_pickle_path_localpath():
     tm.assert_frame_equal(df, result)
 
 
-def test_legacy_sparse_warning(datapath):
+@pytest.mark.parametrize("typ", ["sparseseries", "sparseframe"])
+def test_legacy_sparse_warning(datapath, typ):
```

Comment:
OT we should kill this warning :->

Reply:
Not sure where this warning is coming from (maybe from the pickled object since this is a pre-version 1.0 pickle)